internal/server/sys/os.go

```diff
@@ -8,17 +8,26 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
+	"sync"
+
+	incusAPI "github.com/lxc/incus/v6/shared/api"
+	"github.com/lxc/incus/v6/shared/revert"
 
 	"github.com/FuturFusion/migration-manager/internal/util"
+	"github.com/FuturFusion/migration-manager/shared/api"
 )
 
 // OS is a high-level facade for accessing operating-system level functionalities.
 type OS struct {
+	// A lock to manage filesystem access during uploads.
+	uploadLock sync.Mutex
+
 	// Directories
 	CacheDir string // Cache directory (e.g., /var/cache/migration-manager/)
 	LogDir   string // Log directory (e.g. /var/log/).
 	RunDir   string // Runtime directory (e.g. /run/migration-manager/).
 	VarDir   string // Data directory (e.g. /var/lib/migration-manager/).
+	UsrDir   string // Static directory (e.g. /usr/lib/migration-manager/).
```

Comment:
Based on comment https://github.com/FuturFusion/operations-center/pull/205#issuecomment-3225125447 from @stgraber, this should be changed to `/usr/share/migration-manager/ui`.